elements/lisk-blocks/test/unit/process.spec.ts

```diff
@@ -753,8 +753,8 @@ describe('blocks/header', () => {
 			it('should update vote weight on voted delegate', async () => {
 				const delegateOne = await stateStore.account.get(delegate1.address);
 				const deletateTwo = await stateStore.account.get(delegate2.address);
-				expect(delegateOne.voteWeight).toBe('9889999900');
-				expect(deletateTwo.voteWeight).toBe('9889999900');
+				expect(delegateOne.voteWeight?.toString()).toBe('9889999900');
```

Comment:
since the value we expect is to be bigint, shouldn't the test expectation change to `BigInt('9889999900')`? same for below assertions